src/components/ResetButton.js

```diff
@@ -0,0 +1,18 @@
+import '../css/App.css';
+
+const ResetButton = ({setPointsProp, setStatusProp, setQuoteProp, placeholderQuoteProp}) => {
+    
+    const handleClick = () => {
+        setPointsProp(0);
```

Comment:
This logic here could be the preface of creating a `reset` function in the engine that I mentioned in an earlier comment

Reply:
Cheers, nice shout, I've also done this now!